src/main/java/se/liu/ida/hefquin/engine/queryplan/executable/impl/ops/ExecOpBinaryUnion.java

```diff
@@ -1,28 +1,34 @@
 package se.liu.ida.hefquin.engine.queryplan.executable.impl.ops;
 
 import se.liu.ida.hefquin.engine.data.SolutionMapping;
+import se.liu.ida.hefquin.engine.queryplan.executable.ExecutablePlanVisitor;
 import se.liu.ida.hefquin.engine.queryplan.executable.IntermediateResultBlock;
 import se.liu.ida.hefquin.engine.queryplan.executable.IntermediateResultElementSink;
 import se.liu.ida.hefquin.engine.queryproc.ExecutionContext;
 
 public class ExecOpBinaryUnion implements BinaryExecutableOp{
 
-	@Override
+	@Override
 	public int preferredInputBlockSize() {
-		// Since this algorithm processes the input solution mappings
-		// sequentially (one at a time), an input block size of 1 may
+		// Since this algorithm processes the input solution mappings
+		// sequentially (one at a time), an input block size of 1 may

```

Comment:
What has changed here? It seems nothing.

There are many such lines in this PR that have not actually changed in their content. Perhaps the change is only in the whitespace or the line break character. Please try to avoid such things in the future. That will make my life with the code review easier ;-)